ui/app/AppLayouts/Chat/stores/RootStore.qml

```diff
@@ -55,6 +56,10 @@ QtObject {
         }
     }
 
+    property var channelsModel: SortFilterProxyModel {
+        sourceModel: chatCommunitySectionModule.channelsModel
+    }
```

Comment:
Any reason to wrap it into `SortFilterProxyModel`? :thinking:

Reply:
Nope this is a left over. In fact I can remove the entire `channelsModel` thing